### PR TITLE
Add Sentry. Closes #2

### LINKED ISSUE #2: Sentry

### PR DESCRIPTION
**Pull request by juliangruber**

### CODE REVIEW

**Comment by juliangruber** [comment]
@bajtos hindsight please

File: bin/station-wallet-screening.js [hunk 1]
```diff
@@ -2,11 +2,30 @@ import http from 'node:http'
...
+
+const pkg = JSON.parse(
+  await fs.readFile(
+    fileURLToPath(new URL('../package.json', import.meta.url)),
+    'utf8'
+  )
+)
+
+Sentry.init({
+  dsn: 'https://a12a79e133de58d233f796c6ce0a5072@o1408530.ingest.sentry.io/4506173874307072',
```
Comment (bajtos) [inline bin/station-wallet-screening.js:23]: Sanity check: Did you create a new Sentry project for this?

In the past, we used to copy & paste DSNs and report errors from various services to the same Sentry project, which led to various issues.

**Review by bajtos — COMMENTED** [review]
👍🏻

File: bin/station-wallet-screening.js (hunk 1 above)
Comment (juliangruber) [reply bin/station-wallet-screening.js:23]: Yes, I have created new ones for this, station/withdraw and station/wallet-screening :)

Comment (bajtos) [reply bin/station-wallet-screening.js:23]: Perfect!

**Pull request merged**